mlir/include/mlir/Dialect/Vector/IR/VectorOps.h

```diff
@@ -68,9 +68,13 @@ enum class BroadcastableToResult {
   DimensionMismatch = 2,
   SourceTypeNotAVector = 3
 };
+struct VectorDim {
+  int64_t dim;
+  bool scalableFlag;
+};
```

Comment:
This PR is unrelated to that discussion. I'm only adding this here to avoid adding new set of params to `isBroadcastableTo`.

I believe that before we commit to any new wider API, we should discuss the internal representation of `VectorType` and how scalable dimensions are represented. I am working on a proposal, but that's not yet ready to share 😅 I'm hoping to have something in the coming weeks.